tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/PartialViews.spec.ts

```diff
@@ -0,0 +1,240 @@
+﻿import {AliasHelper, ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+import {expect} from "@playwright/test";
+
+test.describe('Partial Views tests', () => {
+  
+  test.beforeEach(async ({page, umbracoApi, umbracoUi}) => {
+    await page.goto(umbracoApi.baseUrl + '/umbraco');
+    await umbracoUi.goToSection(ConstantHelper.sections.settings);
+  });
+
+  async function gotoPartialViewDetail(page, partialViewFileName) {
+    await page.locator('umb-tree-item').filter({ hasText: 'Partial Views' }).locator('#caret-button').click();
+    await page.locator('umb-tree-item').getByLabel(partialViewFileName).click();
+  }
+  
+
+  test('can create an empty partial view', async ({page, umbracoApi}) => {
+    const partialViewName = 'TestPartialView';
+    const partialViewFileName = partialViewName + ".cshtml";
+    // Act
+    await umbracoApi.partialView.ensureNameNotExistsAtRoot(partialViewName);
```

Comment:
We are using the AAA pattern and ensureNameNotExistsAtRoot is part of Arrange and not Act. This also occurred in some of the other tests